Update/omake_02.txt

```diff
@@ -1726,7 +1726,7 @@ void main()
 	PlaySE(4, "S20/09/440900118", 128, 64);
```

Comment:
Thanks. Keep in mind that the PlaySE should be voice of the line directly *below* it, not above it.